### PR TITLE
Feature/image uploading

### PR DESCRIPTION
**Pull request by dotanthanh**
support image uploading in creating/updating for both customer and person. Proper notification yet to be implemented.

### CODE REVIEW

File: src/constants/customer-form-config.js [hunk 1]
```diff
@@ -33,20 +35,49 @@ export const fields = [
...
+            customersStore.createCustomer(
+              Object.assign({}, form.values(), {logo: logoUrl})
+            );
+          });
+      } else {
+        customersStore.createCustomer(form.values());
+      }
     } else {
-      customersStore.updateCustomer(form.values());
+      if (form.values().file !== '') {
```
Comment (suppayami) [inline src/constants/customer-form-config.js]: `if (form.values().file) {` is enough

Comment (dotanthanh) [reply src/constants/customer-form-config.js]: form from mobx-react-form automatically fills empty value with an empty string, so it is needed.

File: src/customer-view/components/CustomerView.js [hunk 2]
```diff
@@ -42,16 +44,26 @@ class CustomerView extends Component {
       customer => customer.id === this.props.match.params.id
     );
     if (!customer) {
-      return <Loading />;
+      return (
+        <LoadFailedRedirect
+          message={`Timeout, no customer with id "${this.props.match.params
+            .id}" was found`}
```
[the remark this answers is not in the document]
Comment (dotanthanh) [reply src/customer-view/components/CustomerView.js]: fixed

File: src/customers-container/components/CustomerForm.js [hunk 3]
```diff
@@ -1,17 +1,39 @@
...
+import ImageUpload from '../../form-image';
 
 @observer
 class CustomerForm extends Component {
+  @observable previewImage = `http://${this.props.form.$('logo').value}`;
+
   componentWillMount() {
     Modal.setAppElement(document.body);
   }
+  onChangeImage = event => {
```
[the remark this answers is not in the document]
Comment (dotanthanh) [reply src/customers-container/components/CustomerForm.js]: fixed

[the remark this answers is not in the document]
Comment (dotanthanh) [reply src/customers-container/components/CustomerForm.js]: fixed

File: src/person-view/components/PersonView.js [hunk 4]
```diff
@@ -41,7 +42,12 @@ class PersonView extends Component {
       person => person.id === this.props.match.params.id
     );
     if (!person) {
-      return <Loading />;
+      return (
+        <LoadFailedRedirect
+          message={`Timeout, no person with id "${this.props.match.params
```
[the remark this answers is not in the document]
Comment (dotanthanh) [reply src/person-view/components/PersonView.js]: fixed

File: src/people-container/components/PersonForm.js [hunk 5]
```diff
@@ -1,23 +1,45 @@
...
 class PersonForm extends Component {
+  @observable previewImage = `http://${this.props.form.$('picture').value}`;
+
   componentWillMount() {
     Modal.setAppElement(document.body);
     if (skillsStore.skills.length === 0) {
       skillsStore.fetchSkills();
     }
   }
+  onChangeImage = event => {
```
[the remark this answers is not in the document]
Comment (dotanthanh) [reply src/people-container/components/PersonForm.js]: fixed, moved the duplicated func to utility

**Pull request merged**